GlyssenEngineTests/Quote/QuoteParserTests.cs

```diff
@@ -5370,6 +5370,28 @@ public void Parse_MultipleInterruptionsInMiddleOfMultiBlockQuote_MultiBlockQuote
 			Assert.AreEqual(CharacterVerse.kScriptureCharacter, output[5].CharacterId);
 		}
 		#endregion
+
+		[TestCase(19, 10, "6", "narrator-PSA")]
```

Comment:
Sorry I didn't catch this sooner, but these tests should be in a new fixture: ParserCharacterRepositoryTests. Otherwise, it looks as if you're testing QuoteParser.GetCharacters, and there is no such method. The new file should be added to the Quote folder (since the class you're testing is in GlyssenEngine\Quote).